packages/core/content-manager/server/src/history/services/utils.ts

```diff
@@ -9,13 +9,10 @@ import { FIELDS_TO_IGNORE } from '../constants';
  */
 export const getSchemaAttributesDiff = (
   versionSchemaAttributes: CreateHistoryVersion['schema'],
-  contentTypeSchemaAttributes: Schema.Attributes
+  contentTypeSchemaAttributes: Record<string, Schema.Attribute.AnyAttribute>
```

Comment:
you can do Schema.Attributes<UID.Schema> to have the right type

Reply:
Once #19879 is merged, you will be able to use `Schema.Attributes`